models/repo.go

```diff
@@ -1666,6 +1693,125 @@ func GitGcRepos() error {
 		})
 }
 
+// Maybe move this to gogits/git-module
```

Comment:
Make a separate issue for that (when this is merged) instead of adding useless comments in the source-code :wink: